values.yaml

```diff
@@ -184,9 +187,9 @@ zabbixagent:
   resources: {}
   image:
     # -- Zabbix agent Docker image name. Can use zabbix/zabbix-agent or zabbix/zabbix-agent2
-    repository: zabbix/zabbix-agent
+    repository: zabbix/zabbix-agent2
     # -- Tag of Docker image of Zabbix agent
-    tag: ubuntu-6.0.4
+    #tag: ubuntu-6.0.5
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line can be removed, correct?